apps/api/docs/scripts/main-logs-to-excel.py

```
@@ -0,0 +1,44 @@
+        for line in f:
+            try:
+                log_entry = json.loads(line.strip())  # Parse JSON
+
+                # Ensure log_entry is a list (array) or a dictionary (object)
+                if isinstance(log_entry, list):
+                    for entry in log_entry:
+                        if isinstance(entry, dict) and "level" in entry and entry["level"] == "error":
+                            error_message = entry.get("message", "Unknown Error")
+                            error_counts[error_message] += 1
+                elif isinstance(log_entry, dict):
+                    if "level" in log_entry and log_entry["level"] == "error":
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)